jobs/buildhub/utils.py

```diff
@@ -11,6 +11,19 @@
     }
 
 
+def localize_nightly_url(nightly_url):
+    nightly_url = nightly_url.replace('-l10n', '')
+    parts = nightly_url.split('.')
+    locale = 'en-US'
+    index = -3
+    if 'tar' in nightly_url:
+        index = -4
+    if 'mobile' in nightly_url:
+        locale = 'multi'
+    parts[index] = locale
+    return '.'.join(parts)
```

Comment:
I usually prefer regexps, but it may be a matter of taste...

Also could you add a comment here to explain what does this function? At least with an example and why we have to do this